### DISCUSSION TITLE
Permission denied on port 3000 in windows 💣

### DISCUSSION THREAD

**Discussion by fadhilsaheer** [vercel/next.js#24536] [Help] [answered]
Every single time when I try to run `yarn dev`  it will throw me up with this error 😷

```bash
Error: listen EACCES: permission denied 0.0.0.0:3000
    at Server.setupListenHandle [as _listen2] (net.js:1301:21)
    at listenInCluster (net.js:1366:12)
    at doListen (net.js:1503:7)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  code: 'EACCES',
  errno: -4092,
  syscall: 'listen',
  address: '0.0.0.0',
  port: 3000
}
```

I need to restart the entire system to get it working and I'm on windows 🥶

**Comment by unripper** [comment]
I have the same problem, could't find the answer any where 🤷‍♂️

**Comment by fadhilsaheer** [comment, marked as answer]
I got the answer from stackoverflow, It was not a problem from `Next Js` but from `Windows` 😅
 
https://stackoverflow.com/questions/67292618/eacces-permission-denied-0-0-0-03000-in-windows-next-js

Next js should ask to open in another port like react app 🥲
